projects/pgai/pyproject.toml

```diff
@@ -1,29 +1,44 @@
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
```

Comment:
Hatchling allows building from pyproject.toml directly. Setuptools is regarded as legacy tooling. The folks at uv say eventually they will release their own build tool but for now this is their recommendation.